cmd/apply.go

```diff
@@ -97,6 +98,17 @@ func RunApply(options *ApplyOptions) error {
 			BasePath:    options.StateStorePath,
 			ClusterName: name,
 		})
+	case "git":
+		logger.Info("Selected [git] state store")
+		stateStore = git.NewJSONGitStore(&git.JSONGitStoreOptions{
+			BasePath:    options.StateStorePath,
+			ClusterName: name,
+			CommitConfig: &git.JSONGitCommitConfig{
```

Comment:
I think this should be populated with real data. As far as I understand, Name should match committer's name, not cluster's name.
Also e-mail and remote should be populated as well.

Currently, I'm not sure is it really usable without real data.

As I've said earlier, we can just use environmental variables. We already have some functions in the `cmd` pacakge, contained in the [`env.go`](https://github.com/kris-nova/kubicorn/blob/master/cmd/env.go) file, for example `strEnvDef` and `intEnvDef`.
I would just add some sort of function that will fail if it isn't defined instead of providing default value.

Beside environmental variables, I would add flags for defining required values as well.

Also, if you're adding new environmental variables, make sure to document them in the [`envar.md`](https://github.com/kris-nova/kubicorn/blob/master/docs/_documentation/envar.md) file.

Reply:
Thanks for the updates @xmudrii It should not be a hard fix. I'm thinking of using this package: https://github.com/tcnksm/go-gitconfig, If not I could implement it from scratch. Just let me know which works best. I'll meet you on slack to work on this further.

Reply:
@Ch3ck It looks great, you can go with that package.

Reply:
Fixed!